Python-path-traversal-files-main/HFO4/FuckSuperClassTimetable/r.py

```
@@ -0,0 +1,388 @@
+					for x in classLine[-3].split(","):
+						writeClass(className,teacher,room,x,classLine[-2],jsonMode)
+				else:
+					writeClass(className,teacher,classLine[-1],classLine[-3],classLine[-2],jsonMode)
+			else:
+				if("," in classLine[-3]):
+					for x in classLine[-3].split(","):
+						writeClass(className,teacher,room,x,classLine[-2],jsonMode)
+				else:
+					writeClass(className,teacher,classLine[-1],classLine[-3],classLine[-2],jsonMode)
+	if(jsonMode):
+		filename = uid;
```

Ruff (0.13.3)

Statement ends with an unnecessary semicolon

(E703)